advocacy_docs/partner_docs/SIBVisionsGuide/04-ConfiguringTheIntegration.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: 'Configuring the Integration'
+description: 'Walkthrough on configuring Vision X'
+---
+
+Implementing VisionX with EDB Postgres Advanced Server and EDB Postgres Extended requires the following components:
+
+- EDB Postgres Advanced Server
+- EDB Postgres Extended
+- VisionX low-code platform
+- Oracle database
+- Eclipse with VisionX ePlug 
+
+## Prerequisites
+
+- A running EDB Postgres Advanced Server instance
+- A running EDB Postgres Extended instance
+- A running Oracle database instance
+- VisionX low-code platform installed
+- Eclipse with VisionX ePlug installed
+- An Oracle Forms or Oracle APEX application
+
```

Comment:
Don't Add yet.  Need to confirm with Kashif.

Once all components are installed and running you are ready to use Vision X and the appropriate EDB Database.